main/linux-headers/template.py

```diff
@@ -8,7 +8,7 @@
 license = "GPL-2.0-only"
 url = "http://www.kernel.org"
 source = f"$(KERNEL_SITE)/kernel/v{pkgver[0]}.x/linux-{pkgver}.tar.xz"
```

Comment:
Why not use secure protocol i.e. https://www.kernel.org vs http:// ? Yes, there is a sha256 afterwards which is great but probably nice to use HTTPS always if available ?

Reply:
(see `sites` in `src/cbuild/core/template.py` for those values)